addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -103,13 +103,29 @@
             font-size: var(--Card-Footer-font-size, 1rem);
         }
 
-        aside.o_kanban_aside_full > * {
-            height: 100%;
-            @include media-breakpoint-up(md) {
-                height: calc(100% + var(--KanbanRecord-padding-v)* 2);
-                margin-top: calc(var(--KanbanRecord-padding-v)* -1);
-                margin-bottom: calc(var(--KanbanRecord-padding-v)* -1);
-                margin-left: calc(var(--KanbanRecord-padding-h)* -1);
+        aside {
+            flex: 0 0 var(--KanbanRecord__image-width);
+
+            img {
+                height: var(--Card-Aside-Image-height, var(--KanbanRecord__image-width));
```

Comment:
OK, basically we're using `--KanbanRecord` prefix everywhere except for the footer's gap  🤔

I thought that `--KanbanRecord`  was legacy and `--Card` was used for the "new design"... but apparently it's not the case.
I was partially wrong [here](https://github.com/odoo/odoo/pull/169790#discussion_r1645446245), sorry for the headache 🙂

Still, `-KanbanAsideImage-height` isn't a good fit neither, we should replicate the current variable naming IMO:

```
--KanbanRecord__image-width    <- exists already
--KanbanRecord__image-height   <- new
```

What do you think?